dto/src/main/java/org/jboss/pnc/dto/response/MilestoneInfo.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2019 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.time.Instant;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * This is entry describing milestone that produced or consumed an artifact.
+ *
+ * @author Honza Brázdil &lt;jbrazdil@redhat.com&gt;
+ */
+@Data
+@Builder(builderClassName = "Builder")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonDeserialize(builder = MilestoneInfo.Builder.class)
+public class MilestoneInfo {
```

Comment:
Hmm... The name seems is kinda off. You provide much more than Milestone info

Reply:
Well. it provides info about a milestone: In what product and version the milestone is, what is the milestone version, if the milestone was released or not...

Reply:
+1 with Honza, as above a milestone is really product + ver + milestone , it's decomposed purely for efficiency in computer world IMO.

Reply:
Yeah, you are right